src/libs/focusComposerWithDelay.ts

```diff
@@ -20,13 +25,27 @@ function focusComposerWithDelay(textInput: TextInput | null): FocusComposerWithD
 
         if (!shouldDelay) {
             textInput.focus();
+            if (forceSetSelection) {
```

Comment:
I am thinking about a way how to avoid nested `if` blocks. I understand that one nested can be, but avoid the 3rd level.

Reply:
I believe this is the shortest way - the only alternatives I can think of here would be either a one-liner with the ternary operator `(Platform.OS === CONST.PLATFORM.WEB ? (textInput as HTMLTextAreaElement).setSelectionRange : (textInput as TextInput).setSelection)(forceSetSelection.start, forceSetSelection.end)` (which would be unreadable) or extracting the `focus()` and `setSelection` logic to a local function - however I don't like that idea. Do you think as-is is the best option or we should go with one of these?

Reply:
I got you, agree. I prefer the nested `if` blocks to the one-line solution.  👍